### PR TITLE
Update thriftrw to 2.2.4

### PR DESCRIPTION
**Pull request by kriskowal**

### CODE REVIEW

**Comment by kriskowal** [comment]
This addresses an issue with encoding/decoding sets.

r @jcorbin @Raynos 
cc @Willyham

File: node/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "safe-json-parse": "^4.0.0",
     "sse4_crc32": "3.2.0",
     "tape-cluster": "2.1.0",
-    "thriftrw": "2.2.3",
+    "thriftrw": "^2.2.4",
```
Comment (jcorbin) [inline node/package.json]: why caret

**Comment by kriskowal** [comment]
This time with pinned version

**Comment by jcorbin** [comment]
:+1:

**Pull request merged**